scripts/fetch_aic_models.py

```
@@ -0,0 +1,73 @@
+os.makedirs(DEST_DIR, exist_ok=True)
+
+for model_name in models:
+    blob_path = f"{prefix}/{model_name}" if prefix else model_name
+    dest = f"{DEST_DIR}/{model_name}"
+    try:
+        bucket = client.bucket(bucket_name)
+        blob = bucket.blob(blob_path)
+        blob.download_to_filename(dest)
+        size = os.path.getsize(dest)
+        print(f"Fetched {model_name}: {size} bytes")
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)